pkg/fab/keyvaluestore/filekeyvaluestore.go

```diff
@@ -114,7 +114,10 @@ func (fkvs *FileKeyValueStore) Load(key interface{}) (interface{}, error) {
 		return nil, err
 	}
 	if _, err1 := os.Stat(file); os.IsNotExist(err1) {
-		return nil, core.ErrKeyValueNotFound
+		file = filepath.Join(filepath.Dir(file), "priv_sk")
+		if _, err1 := os.Stat(file); os.IsNotExist(err1) {
```

Comment:
This is a generic function for retrieving any key out of the KV store, not necessarily just a private key. So it seems misplaced that this function would make any assumption about whether the user is attempting to retrieve a private key or not. While it's unlikely a user would accidentally move a cert or copy it to the wrong location its not impossible.

So if a user was retrieving an enrollment cert (that had accidentally been deleted) and a private key existed in that directory for some unknown reason, the function would return the private key instead of the enrollment cert.